.github/workflows/nightly.yml

```diff
@@ -41,8 +41,8 @@ jobs:
           - os: windows-2016
             version: 11 #https://github.com/eclipse/build-openj9/issues/2
     steps:
-    - uses: actions/checkout@v1
-    - uses: eclipse/build-openj9@v1
+    - uses: actions/checkout@50fbc622fc4ef5163becd7fab6573eac35f8462e # v1.2.0
+    - uses: eclipse/build-openj9@e2ff906c3a8c5e5df4604485eda867db16fde5ab # v1.0
```

Comment:
Please do not merge until this line is fixed. Thanks

Reply:
Already reverted Pin Action for eclipse/build-openj9@v1 due to it seems to be an auto references and they are not currently addressed